### ISSUE TITLE
Focus on mouseover not working

### ISSUE THREAD

**Issue by chalkos** [chabou/hyper-pane#42]
Hi, the config `config.paneNavigation.focusOnMouseHover` stopped working on 3.0.0-canary.5

A warning on the console reads:

```
The underlying terminal engine of Hyper no longer uses iframes with individual `document` objects for each terminal instance. This method call is retained for backwards compatibility reasons. It's ok to attach directlyto the `document` object of the main `window`.
    getTermDocument @ bundle.js:1
```

**Comment by chabou** [comment]
Thank you for your feedback 🙏 
I reproduced your issue.

It seems due to some breaking change with: `term.termRef`
https://github.com/chabou/hyper-pane/blob/7f0c766c5d72bb1d294af462a50a896b5672a563/index.js#L629-L637

Old fashioned way is wrongly used

**Comment by chabou** [comment]
Fixed in Hyper v3.0.0-canary.6 🎉

**Comment by chalkos** [comment]
It is working for me too, thanks! :tada:
